src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+---
+title: "Seamless Data Movement Across Clusters using RBD Live Migration in Squid"
+
+date: 2025-02-07
+author: Sunil Angadi, Anthony D'Atri (IBM)
+tags:
+  - ceph
+  - rbd
+---
+
+## Seamless Data Movement Across Clusters using RBD Live Migration in Squid
+
+### Introduction to Block Storage
+
+A block is a small, fixed-sized piece of data, like a 512-byte chunk. Imagine
+breaking a book into many pages: each page is a "block". When you put all the
+pages together, you get a complete book, just like combining many smaller data
+blocks creates a larger storage unit.
+
+#### Where Do We Use Blocks?
+
+Block-based storage is commonly used in computing, and in devices including:
+* Hard drives or SSDs (your computer's storage)
+* CDs/DVDs (discs you use for music or movies)
+* Floppy disks (older storage devices)
+* Tape drives (used in big data backup systems)
+
+#### What Are Ceph RADOS Block Devices (RBD)?
+
+Ceph Rados Block Devices are a type of storage served by a Ceph cluster that
+works like a virtual physical storage drive. Instead of storing data on a single
+device, it spreads the data across multiple storage nodes and devices (called
+OSDs) in a Ceph cluster. This makes it efficient, scalable, and reliable.
+
+#### Why Are Ceph Block Devices Special?
+
+Ceph Block Devices have some amazing features:
+* Snapshots – Like taking a photo of your data at a moment in time, so you can restore or roll back to it later.
+* Replication – Your data is copied multiple times to prevent loss.
+* Data consistency – Ensures your data is accurate and safe.
+
+#### How Do Ceph Block Devices Work?
+
+Clients use the ``librbd`` library to talk to the components of a Ceph
+cluster and manage data efficiently.
+
+#### Who Uses Ceph Block Devices?
+
+* Cloud computing platforms like OpenStack, Proxmox, AWS EBS, and IBM Cloud rely on them.
+* Virtual machines (KVM/QEMU) use them for fast and scalable storage.
+* They can even work alongside Ceph’s Object Storage for a flexible storage solution.
+
+In short, Ceph Block Devices provide fast, scalable, and reliable storage for
+modern computing needs, ensuring that data is always available, accurate,
+and safe.
+
+## Ceph RBD Live Migration of Images
+
+As a storage administrator, you have the power to seamlessly move (_live-migrate_)
+RBD images within your Ceph cluster or to a different Ceph cluster. Think of this
+like moving a file from one folder to another on your computer, but in this case,
+it happens withinin or between large, distributed storage clusters.
+
+Note that RBD images are also known as _volumes_, a term that is less likely to
+be confused with graphics files containing the likeness of Taylor Swift or cats.
```

Comment:
Yeah, why not?

Reply:
Surely the cardinality of these in the world's object stores must be staggering.